### PR TITLE
[TIC-665] Library updates

### PR DESCRIPTION
**Pull request by itailevi98**
https://propelauth.atlassian.net/browse/TIC-665

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,15 +12,16 @@
         "user"
     ],
     "dependencies": {
-        "jsonwebtoken": "^9.0.0"
+        "jose": "^5.2.0",
+        "jsonwebtoken": "^9.0.2"
```
Comment (andrew-propelauth) [inline package.json:16]: jsonwebtoken should be removed as a dependency

File: src/user.ts [hunk 2]
```diff
@@ -99,20 +256,28 @@ export class OrgMemberInfo {
     }
 
     public hasAllPermissions(permissions: string[]): boolean {
-        return permissions.every(permission => this.hasPermission(permission))
-    }
-
-    public get assignedRole(): string {
```
Comment (andrew-propelauth) [inline src/user.ts:105]: Need to keep these for backwards compatibility

**Review by andrew-propelauth — COMMENTED** [review]
two small comments then lgtm

**Review by pfvatterott — COMMENTED** [review]
works for me with testing!

**Review by andrew-propelauth — APPROVED** [review]
lgtm

**Pull request merged**